### DISCUSSION TITLE
Re-enabeling Repositories after Upgrade?

### DISCUSSION THREAD

**Discussion by michaelmariaott** [SpiralLinux/SpiralLinux-project#277] [Q&A]
Hello,
I followed the guide in the documentation to upgrade a fresh spiral linux install to Debian 12 Bookworm. So the upgrade went through without any error messages (whenever I was asked wether to  keep or replace a config file I kept the original one, after reading through the changes). Now I am wondering which software repositories I should (re-)enable after the upgrade. I obviously enabled everything the guide told me to. But on a fresh spiral linux install, the Debian, Fast Track and Fast Track Staging Repos where enabled. Should I rename them to bookworm and enable them again?
Thank you for your excellent work on spiral linux!

![SoftwareUpdates](https://github.com/SpiralLinux/SpiralLinux-project/assets/87768914/7a0f7033-083c-40e9-81f5-14e1535ed10b)

**Comment by zparihar** [comment]
Fast Track for VirtualBox is not ready for Bookworm yet.  That's why Spiral Linux Bookworm has not been release.

I'm trying to follow up with Virtualbox:

https://www.virtualbox.org/ticket/21524

**Comment by FabioLolix** [comment]
Question from an outsider, no one can rebuild the virtualbox .deb package on Bookworm? The build files (.dsc, .orig.tar.xz) are public or not?

**Comment by zparihar** [comment]
Its more so about including the 'fasttrack' repo in Spiral Linux, where that repo has a bookworm deb available.

Zubin Singh Parihar
***@***.*** ***@***.***/0?redirect=mailto%3Azparihar%40gmail.com&recipient=cmVwbHkrQUFGSERXWDNEM1Y0U1BHRFlERTZCSU9DVkdNM0pFVkJNWEhBQVVMWVY0QHJlcGx5LmdpdGh1Yi5jb20%3D)
+1.604.767.7900 ***@***.***/1?redirect=tel%3A%2B1.604.767.7900&recipient=cmVwbHkrQUFGSERXWDNEM1Y0U1BHRFlERTZCSU9DVkdNM0pFVkJNWEhBQVVMWVY0QHJlcGx5LmdpdGh1Yi5jb20%3D)
http://ca.linkedin.com/in/zubinparihar ***@***.***/2?redirect=http%3A%2F%2Fca.linkedin.com%2Fin%2Fzubinparihar&recipient=cmVwbHkrQUFGSERXWDNEM1Y0U1BHRFlERTZCSU9DVkdNM0pFVkJNWEhBQVVMWVY0QHJlcGx5LmdpdGh1Yi5jb20%3D)

**Comment by straffer44** [comment]
To the OP double check your recommended  and security updates.  They are for source code and not binary.  Also check (with edit button) that they are on bookworm and not bullseye.  This happened to me when I upgraded sprial linux.  Don't know if it was something I did but the "other software" tab shows bullseye.  When I edited recommended  and security to bookworm they both disappeared form the list.  I then started to get updates tho.

**Comment by bemetustitu** [comment]
thank you, good info
